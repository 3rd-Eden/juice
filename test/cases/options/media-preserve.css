@media screen {
	p {
		color: blue !important;
		background: red;
	}
}
