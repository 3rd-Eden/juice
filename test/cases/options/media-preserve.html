<html>
<head>
    <style>
    p {
    color: blue;
    }
    @media screen {
    a {
    color: blue !important;
    background: red;
    }
    }
    </style>
</head>
<body>
    <p>hi</p>
</body>
</html>
