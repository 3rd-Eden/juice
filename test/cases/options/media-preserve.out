<html>
<head>
    <style>
@media screen {
  a {
    color: blue !important;
    background: red;
  }
}
</style>
</head>
<body>
    <p style="color: blue;">hi</p>
</body>
</html>
